location/src/main/java/org/odk/collect/location/GoogleFusedLocationClient.kt

```diff
@@ -0,0 +1,142 @@
+package org.odk.collect.location
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.location.Location
+import android.location.LocationManager
+import android.os.Looper
+import com.google.android.gms.location.FusedLocationProviderClient
+import com.google.android.gms.location.LocationCallback
+import com.google.android.gms.location.LocationListener
+import com.google.android.gms.location.LocationRequest
+import com.google.android.gms.location.LocationResult
+import com.google.android.gms.location.LocationServices
+import org.odk.collect.location.LocationClient.LocationClientListener
+import org.odk.collect.location.LocationUtils.sanitizeAccuracy
+
+/**
+ * An implementation of [LocationClient] that uses Google Play Services to retrieve the user's location.
+ *
+ * Should be used whenever Google Play Services is present. In general, use
+ * [LocationClientProvider] to retrieve a configured [LocationClient].
+ */
+class GoogleFusedLocationClient(
```

Comment:
Did any changes get made to `GoogleFusedLocationClient` or its test? Because the Kotlin version happens in the same commit, the diff doesn't give me any insight into that.